src/hindi.js

```diff
@@ -1,13 +1,35 @@
 const allInfo = {
-  tense: ['present', 'past', 'future'],
-  gender: ['m', 'f'],
-  formal: [true, false],
-  wordType: ['adjective', 'verb']
+  Tense: ['Present', 'Past', 'Future'],
```

Comment:
is there a need to capitalize these values? It could be all done through css, and I feel that we are making these changes for UI purposes, so css would be a better solution than capitalizing these values.